alwaysontop/helpers.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Validates a window opened via window.open
+ *
+ * @returns {boolean} - Validity of the window
+ */
+const isValidWindow = win => win && !win.closed;
```

Comment:
Could you please define the functions as `function ...` and not as `const ...`? This way it will be more consistent with the code from jitsi-meet.